### ISSUE TITLE
load scheduler to resume training

### ISSUE THREAD

**Issue by homelifes** [huggingface/accelerate#137]
Hello, is this the correct way to load a scheduler to resume training?

```
# load the model and optimizer first (i've already unwrapped first and used accelerator.save to save the model and optimizer), 
# then, we send to the prepare method:
model, optimizer, train_loader = accelerator.prepare(model, optimizer, train_loader)

# re-define the scheduler first and then load its state dictionary
t_total = len(train_loader) * num_train_epochs
warmup_steps = 0.10 * t_total
scheduler = get_linear_schedule_with_warmup(optimizer, num_warmup_steps=warmup_steps, num_training_steps=t_total)
scheduler.load_state_dict(torch.load(ckpt_path)['scheduler'])
```
Thanks in advance

**Comment by sgugger** [comment]
If you saved it in the "scheduler" of your checkpoint, yes.

Also please use the [forums](https://discuss.huggingface.co/) for questions like this, we keep the issues for bugs and feature requests only.

**Comment by homelifes** [comment]
Thank you and sorry for posting it here.
